Cargo.toml

```diff
@@ -34,18 +34,18 @@ unstable = [] # used for benchmarks
 test_headless = []  # used for testing headless display
 
 [dependencies.glutin]
-version = "0.12"
+version = "0.13"
 features = []
 optional = true
 
 [dependencies]
-backtrace = "0.3.2"
```

Comment:
Are these necessary? It doesn't seem like we need to force users of `glutin` to upgrade backtrace/fnv if they don't need to - and if we leave the old version, users are free to use a wider range of compatible versions.